src/main/java/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package strategy.impl;
+
+import strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
Ensure that your class follows the Single Responsibility Principle. The `ReturnOperation` class seems to be focused on a single task which is good, but make sure it doesn't handle more than it should as your project grows.